test/functional/test_node_ports.py

```diff
@@ -144,3 +145,303 @@ def test_start_node1_then_start_node2_no_conflict_ports(self):
         assert "test1" in ipmi_result and "test2" in ipmi_result
         assert "test1-node" in qemu_result and "test2-node" in qemu_result
 
+    def test_start_node1_then_start_node2_bmc_connection_port(self):
```

Comment:
How about `test_start_nodes_with_conflict_bmc_connection_port()`, better then the foolish name I used in command line test.

So are other tests.